internal/usecases/reports/get_assets_report.go

```diff
@@ -0,0 +1,153 @@
+package reports
+
+import (
+	"context"
+	"github.com/rsksmart/liquidity-provider-server/internal/entities"
+	"github.com/rsksmart/liquidity-provider-server/internal/entities/blockchain"
+	"github.com/rsksmart/liquidity-provider-server/internal/entities/liquidity_provider"
+	"github.com/rsksmart/liquidity-provider-server/internal/entities/quote"
+	"github.com/rsksmart/liquidity-provider-server/pkg"
```

Comment:
this import is from an external layer, remember you should only import things from the same layer or inner ones